.travis.yml

```diff
@@ -28,7 +28,7 @@ script:
   - yarn run build
   - yarn run package
   - zip -r release.zip release -q
-
+  - electron-installer-dmg release/Mac-64bit/Falcon\ SQL\ Client-darwin-x64/Falcon\ SQL\ Client.app mac-falcon-sql-client.dmg
```

Comment:
Not 💯 , but last time I tried something similar.. I was getting errors due to `\` in the spaces between the directory names. 
We may need to add a fix here, by trying * notation for ex:
```
electron-installer-dmg release/Mac-64bit/Falcon*/Falcon*.app mac-falcon-sql-client
```

On a different note, Travis CI queues all open source/free builds together so it takes a lot of time for builds during peak hours (EST mornings, as I have observed recently).

So will be some time before we know if it is working.